staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -36,21 +41,83 @@ var (
 		},
 		[]string{"transformation_type"},
 	)
+	transformerFailuresTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "transformation_failures_count",
```

Comment:
prometheus naming convention for counters is the `_total` suffix, instead of `_count`: https://prometheus.io/docs/practices/naming/

Reply:
Renamed to _total.